definitions/pipelines/immuno.cwl

```diff
@@ -93,191 +306,151 @@ steps:
         in:
             reference: reference
             bams: normal_bams
             readgroups: normal_readgroups
-            mills:
-            known_indels:
-            dbsnp_vcf:
-            bqsr_intervals:
-            bait_intervals:
-            target_intervals:
-            per_base_intervals:
-            per_target_intervals:
-            summary_intervals:
-            omni_vcf:
-            picard_metric_accumulation_level:
-            emit_reference_confidence:
-            gvcf_gq_bands:
-            intervals:
-            vep_cache_dir:
-            synonyms_file:
-            annotate_coding_only:
-            custom_gnomad_vcf:
-            qc_minimum_mapping_quality:
-            qc_minimum_base_quality:
-            custom_clinvar_vcf:
-            optitype_name:
+            mills: mills
+            known_indels: known_indels
+            dbsnp_vcf: dbsnp_vcf
+            bqsr_intervals: bqsr_intervals
+            bait_intervals: bait_intervals
+            target_intervals: target_intervals
+            per_base_intervals: per_base_intervals
+            per_target_intervals: per_target_intervals
+            summary_intervals: summary_intervals
+            omni_vcf: omni_vcf
+            picard_metric_accumulation_level: picard_metric_accumulation_level
+            emit_reference_confidence: emit_reference_confidence
+            gvcf_gq_bands: gvcf_gq_bands
+            intervals: gatk_haplotypecaller_intervals
+            vep_cache_dir: vep_cache_dir
+            synonyms_file: synonyms_file
+            annotate_coding_only: annotate_coding_only
+            custom_gnomad_vcf: custom_gnomad_vcf
+            qc_minimum_mapping_quality: qc_minimum_mapping_quality
+            qc_minimum_base_quality: qc_minimum_base_quality
+            custom_clinvar_vcf: custom_clinvar_vcf
+            optitype_name: optitype_name
+            vep_assembly: vep_assembly
         out:
-            [final_vcf]
+            [cram,mark_duplicates_metrics,insert_size_metrics,insert_size_histogram,alignment_summary_metrics,hs_metrics,per_target_coverage_metrics,per_target_hs_metrics,per_base_coverage_metrics,per_base_hs_metrics,summary_hs_metrics,flagstats,verify_bam_id_metrics,verify_bam_id_depth,gvcf,final_vcf,coding_vcf,limited_vcf,vep_summary,optitype_tsv,optitype_plot]
     phase_vcf:
         run: ../subworkflows/phase_vcf.cwl
         in:
             somatic_vcf: somatic/final_vcf
             germline_vcf: germline/final_vcf
             reference: reference
-            reference_dict:
+            reference_dict: reference_dict
             bam: somatic/tumor_cram
         out:
             [phased_vcf]
     pvacseq:
-        run: pvacseq.cwl
+        run: ../subworkflows/pvacseq.cwl
         in:
             detect_variants_vcf: somatic/final_vcf
+            sample_name: sample_name #should this be the same as the above sample_name
+            normal_sample_name: normal_name #is this right?
```

Comment:
These needs to be normal sample name in the pVACseq input VCF. I believe in our pipelines this ends up being NORMAL.